modules/doc/content/getting_started/installation/hpc_install_moose.md

```diff
@@ -18,30 +18,24 @@ Your cluster will most likely have these two requirements available via some for
 
 ```bash
 export STACK_SRC=`mktemp -d /tmp/stack_temp.XXXXXX`
```

Comment:
remove this as well, since it was only here as a temporary spot to build PETSc.

Reply:
also remove line 52 below, where we include the manual clean up (nothing to clean up now)